extensions/skyportal/skyportal/handlers/api/kowalski_filter.py

```diff
@@ -50,46 +59,42 @@ def get(self, filter_id):
                 application/json:
                   schema: Error
         """
-        if filter_id is not None:
-            f = (
-                DBSession()
-                .query(Filter)
-                .filter(
-                    Filter.id == filter_id,
-                    Filter.group_id.in_(
-                        [g.id for g in self.current_user.accessible_groups]
-                    ),
-                )
-                .first()
-            )
-            if f is None:
-                return self.error("Invalid filter ID.")
-
-            group_id = f.group_id
-
-            base_url = (
-                f"{self.cfg['app.kowalski.protocol']}://"
-                f"{self.cfg['app.kowalski.host']}:{self.cfg['app.kowalski.port']}"
+        f = (
+            DBSession()
+            .query(Filter)
+            .filter(
+                Filter.id == int(filter_id),
+                Filter.group_id.in_(
```

Comment:
Should happen in a separate PR, but Fritz extensions need to be updated to use the new access control infrastructure

Reply:
Right, good point